doc/GettingStarted.rst

```diff
@@ -11,19 +11,16 @@ All amounts are represented in the smallest unit (eg. cents), so USD 5.00 is wri
    // or shorter:
    $fiver = Money::USD(500);
 
-Autoloading
------------
+Installation
+------------
 
-You'll need an autoloader. Money is PSR-0 compatible, so if you are using the Symfony2 autoloader, this will do:
+Install the library using composer. Add the following to your composer.json:
+
+.. code-block:: json
+
+{
+    "require": {
+        "mathiasverraes/money": "~3.0"
```

Comment:
Copied from readme. I believe current standards indicate we should use cli: composer require. And then composer its preference for versioning. I'll update both.